COPYING

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) <2014> <lumien>
```

Comment:
Please replace line with `Copyright (c) <2014> <[https://github.com/lumien231](lumien231)>` and its good to go